.github/workflows/build.yml

```diff
@@ -42,28 +42,36 @@ jobs:
           echo "ASSETS_HASH=$(ls assets*.tar.co)" >> $GITHUB_ENV
           echo "ASSETS_LOCATION=s3://${BASEMAPS_BUCKET}/assets/${ASSETS_HASH}" >> $GITHUB_ENV
 
-      - name: (Prod) AWS Configure
+      - name: (Build) AWS Configure
         uses: aws-actions/configure-aws-credentials@v1.6.1
         with:
           aws-region: ap-southeast-2
           mask-aws-account-id: true
-          role-to-assume: ${{ secrets.AWS_ROLE_PROD }}
+          role-to-assume: ${{ secrets.AWS_SCREENSHOT_ROLE }}
 
-      - name: (Prod - Config) Bundle Config File
+      - name: Bundle Config File
```

Comment:
This might be broken. I think `bundle` cli need to read dynmodb.

Reply:
bundle CLI does not need access to dynamodb, it only reads the imagery out of s3.